frontend/src/features/projects/common/forms/ApprovalConfirmationForm.tsx

```diff
@@ -1,18 +1,22 @@
 import React, { Fragment } from 'react';
 import { Form } from 'react-bootstrap';
-import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
 import { IStepProps } from '../interfaces';
 import { Check } from 'components/common/form/Check';
+import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
+import Roles from 'constants/roles';
 
 /**
  * Form component of ApprovalConfirmationForm.
  * @param param0 isReadOnly disable editing
  */
 const ApprovalConfirmationForm = ({ isReadOnly }: IStepProps) => {
-  const userDisplayName = useKeycloakWrapper().displayName || 'Pims User';
   const fieldName = 'confirmation';
-  const label = `I, ${userDisplayName}, confirm by checking this box that I have received the approval 
-  to request the property be submitted to the Enhanced Referral Program and marketed internally to other Ministry 
+  const keycloak = useKeycloakWrapper();
+  const isSres = keycloak.hasRole(Roles.SRES) || keycloak.hasRole(Roles.SRES_FINANCIAL_MANAGER);
+  const groupName = isSres ? 'The owning Ministry/Agency' : 'My Ministry/Agency';
```

Comment:
I'm sure a better way to do this would be to check if the project agency matches the viewers agency, but I'm not sure how to access the project agency from this form AND as far as I'm aware the only people outside an agency who would be viewing a project is SRES

Reply:
This works